src/uct/api/v2/uct_v2.h

```diff
@@ -693,8 +693,9 @@ typedef struct {
     size_t            exported_mkey_packed_size;
 
     /**
-     * Global identifier - unique device identifier in a system (can be MAC
-     * address or GUID)
+     * Byte array which hold global unique device identifier (for example, can
```

Comment:
Byte array that holds a globally unique device identifier (for example, a MAC address or a GUID). If global ...

Reply:
done